lib/internal/Magento/Framework/DB/Logger/LoggerProxy.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\DB\Logger;
+
+class LoggerProxy extends LoggerAbstract
+{
+    /**
+     * Logger alias param name
+     */
+    const PARAM_ALIAS = 'db_logger_alias';
```

Comment:
per comment earlier please rename the value of the param from "db_logger_alias" to "db_logger_mode" or something similar.